### ISSUE TITLE
cran build failed when

### ISSUE THREAD

**Issue by shajoezhu** [insightsengineering/rtables#728]
```
* using log directory 'd:/RCompile/CRANincoming/R-devel/rtables.Rcheck'
* using R Under development (unstable) (2023-08-28 r85029 ucrt)
* using platform: x86_64-w64-mingw32
* R was compiled by
    gcc.exe (GCC) 12.2.0
    GNU Fortran (GCC) 12.2.0
* running under: Windows Server 2022 x64 (build 20348)
* using session charset: UTF-8
* checking for file 'rtables/DESCRIPTION' ... OK
* this is package 'rtables' version '0.6.3'
* package encoding: UTF-8
* checking CRAN incoming feasibility ... [15s] NOTE
Maintainer: 'Joe Zhu <joe.zhu@roche.com>'

New maintainer:
  Joe Zhu <joe.zhu@roche.com>
Old maintainer(s):
  Gabriel Becker <gabembecker@gmail.com>
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking serialization versions ... OK
* checking whether package 'rtables' can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking for future file timestamps ... OK
* checking 'build' directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking whether startup messages can be suppressed ... OK
* checking use of S3 registration ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... [30s] OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd line widths ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking installed files from 'inst/doc' ... OK
* checking files in 'vignettes' ... OK
* checking examples ... [52s] OK
* checking for unstated dependencies in 'tests' ... OK
* checking tests ... [182s] OK
  Running 'testthat.R' [182s]
* checking for unstated dependencies in vignettes ... NOTE
'::' or ':::' import not declared from: 'forcats'
* checking package vignettes in 'inst/doc' ... OK
* checking re-building of vignette outputs ... [94s] ERROR
Error(s) in re-building vignettes:
--- re-building 'advanced_usage.Rmd' using rmarkdown
--- finished re-building 'advanced_usage.Rmd'

--- re-building 'baseline.Rmd' using rmarkdown
--- finished re-building 'baseline.Rmd'

--- re-building 'clinical_trials.Rmd' using rmarkdown
--- finished re-building 'clinical_trials.Rmd'

--- re-building 'custom_appearance.Rmd' using rmarkdown
--- finished re-building 'custom_appearance.Rmd'

--- re-building 'example_analysis_coxreg.Rmd' using rmarkdown
--- finished re-building 'example_analysis_coxreg.Rmd'

--- re-building 'exploratory_analysis.Rmd' using rmarkdown

Quitting from lines 145-149 [unnamed-chunk-13] (exploratory_analysis.Rmd)
Error: processing vignette 'exploratory_analysis.Rmd' failed with diagnostics:
there is no package called 'forcats'
--- failed re-building 'exploratory_analysis.Rmd'

--- re-building 'format_precedence.Rmd' using rmarkdown
--- finished re-building 'format_precedence.Rmd'

--- re-building 'introduction.Rmd' using rmarkdown
--- finished re-building 'introduction.Rmd'

--- re-building 'manual_table_construction.Rmd' using rmarkdown
--- finished re-building 'manual_table_construction.Rmd'

--- re-building 'sorting_pruning.Rmd' using rmarkdown
--- finished re-building 'sorting_pruning.Rmd'

--- re-building 'split_functions.Rmd' using rmarkdown
--- finished re-building 'split_functions.Rmd'

--- re-building 'subsetting_tables.Rmd' using rmarkdown
--- finished re-building 'subsetting_tables.Rmd'

--- re-building 'tabulation_concepts.Rmd' using rmarkdown
--- finished re-building 'tabulation_concepts.Rmd'

--- re-building 'tabulation_dplyr.Rmd' using rmarkdown
--- finished re-building 'tabulation_dplyr.Rmd'

--- re-building 'title_footer.Rmd' using rmarkdown
--- finished re-building 'title_footer.Rmd'

SUMMARY: processing the following file failed:
  'exploratory_analysis.Rmd'

Error: Vignette re-building failed.
Execution halted

* checking PDF version of manual ... [28s] OK
* checking HTML version of manual ... [23s] OK
* DONE
Status: 1 ERROR, 2 NOTEs

**Comment by shajoezhu** [comment]
adding `forcats` to suggests resolve the issue.

**Comment by shajoezhu** [comment]
closed in #726
